odata-data-aggregation-ext/3.1 Fundamentals of Input and Output Sets.md

```diff
@@ -48,7 +48,7 @@ Here is an overview of the structural changes made by different transformations:
 - During [join](#Transformationsjoinandouterjoin), one instance with a collection of related instances is replaced by many copies, each of which is related via a dynamic property to one of the related instances.
 - During [concatenation](#Transformationconcat), the same instances are transformed multiple times and the output sets with their potentially different structures are concatenated.
 
-An output set thus consists of instances with different structures. This is the same situation as with a collection of an open type [OData-CSDL, sections "Open Entity Type"]($$$OData-CSDLJSON$$$#OpenEntityType) and ["Open Complex Type"]($$$OData-CSDLJSON$$$#OpenComplexType) and it is handled in the same way.
+An output set thus consists of instances with different structures. This is the same situation as with a collection of an open type ([OData-CSDL, sections "Open Entity Type"]($$$OData-CSDLJSON$$$#OpenEntityType) and ["Open Complex Type"]($$$OData-CSDLJSON$$$#OpenComplexType)) and it is handled in the same way.
```

Comment:
```suggestion
An output set thus consists of instances with different structures. This is the same situation as with a collection of an open type ([OData-CSDL, section "Open Entity Type"]($$$OData-CSDLJSON$$$#OpenEntityType) and [OData-CSDL, section "Open Complex Type"]($$$OData-CSDLJSON$$$#OpenComplexType)) and it is handled in the same way.
```
This is about citation style: [section y] is not a valid citation, [SpecID, section y] is.